desktop/release-notes.md

```diff
@@ -23,6 +23,92 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.11.0
+2022-07-28
+
+> Download Docker Desktop
+>
+> [Windows](https://desktop.docker.com/win/main/amd64/83496/Docker%20Desktop%20Installer.exe) |
```

Comment:
The download links of the latest version were always the general download links without the build number, but also with an UTM. Do we want to continue with this behaviour or have we decided to change this?

When creating a new release notes PR we typically had to change the section of the previous version (line 118) to have the fixed links with the build number in it. I now also realise that line 118 is just an include. 🤔 

The current preview https://deploy-preview-15212--docsdocker.netlify.app/desktop/release-notes/ shows that 4.11.0 looks more compact, then 4.10.1 is more detailed with bigger boxes for each platform, then 4.10.0 is more compact again with the checksums.

So for the the 4.11.0 release I suggest to update the `_include/desktop-install.html` file to (1) update the checksums for 4.11.0, and (2) only update the version of the Linux download links. And use this include here instead of lines 31-59.
For the previous 4.10.1 release section we need to either copy the contents of the include file or make it look like 4.10.0 but with the fixed download links including the build numbers and corresponding checksums.

@aevesdocker What do you recommend how it should look like?

Reply:
Yes you're right @StefanScherer in your recommendation of what needs to be updated so that 4.11.0 is more detailed with bigger boxes and so it stands out more. 

It is slightly more labour-intensive now to produce the release notes, but it's important so we can include the checksums in the most efficient/visually appealing way.

Reply:
I've updated it so that 4.11.0 uses the new layout, and 4.10.1 uses a layout that matches 4.10.0 as discussed.

We should refactor it before the next release so that multiple releases can share `desktop-install.html` if possible.

Reply:
Looks good now, thanks @gavin-timbrell !